jenkinsfiles/Windows_Check.groovy

```diff
@@ -303,8 +303,16 @@ pipeline {
                                                         -DCMAKE_CXX_COMPILER=dpcpp^
                                                         -DONEDPL_BACKEND=dpcpp^
                                                         -DONEDPL_DEVICE_TYPE=GPU ..
+                                                    EXIT /b %ERRORLEVEL%
+                                                """, label: "Generate"
+                                                bat script: """
+                                                    set MAKE_PROGRAM=%DevEnvDir%CommonExtensions\\Microsoft\\CMake\\Ninja\\ninja.exe
                                                     "%MAKE_PROGRAM%" build-all -v -k 0
+                                                    EXIT /b %ERRORLEVEL%
```

Comment:
Are you sure that explicit exit with errorlevel is required here? I'd expect that `bat script` section should handle it implicitly (fail is the latest errorlevel is not 0). Please double-check.